mersenne-31/src/complex.rs

```diff
@@ -33,6 +35,18 @@ impl<AF: AbstractionOf<Mersenne31>> Mersenne31Complex<AF> {
     pub fn imag(&self) -> AF {
         self.parts[1].clone()
     }
+
+    pub fn conjugate(&self) -> Self {
+        Self::new(self.real(), self.imag().neg())
+    }
+
+    pub fn magnitude(&self) -> AF {
+        self.real() * self.real() + self.imag() * self.imag()
+    }
+
+    pub fn scale(&self, factor: AF) -> Self {
```

Comment:
We also overload `*` for complex * base (required by `AbstractExtensionField`). Maybe we should do `/` as well so the inverse can be `self.conjugate() / self.magnitude()`

Reply:
latest push: eliminated `scale` and changed to a `*`
I wouldn't use `/` here just because I switched to getting the `Option` from the base field `try_inverse()`